v23.1/create-database.md

```diff
@@ -24,7 +24,7 @@ To create a database, the user must be a member of the `admin` role or must have
 Parameter | Description
 ----------|------------
 `IF NOT EXISTS` | Create a new database only if a database of the same name does not already exist; if one does exist, do not return an error.
-`name` | The name of the database to create, which [must be unique](#create-fails-name-already-in-use) and follow these [identifier rules](keywords-and-identifiers.html#identifiers).
+`name` | The name of the database to create, which [must be unique](#create-fails-name-already-in-use) and follow these [identifier rules](keywords-and-identifiers.html#identifiers).<br /><br />Do not begin your database name with the string `cluster:`. If your database name begins with this string, you must append the following to the URI connection string to [connect to the cluster](connect-to-the-database.html): `&options=-ccluster=system`
```

Comment:
Instead of "do not begin" I would rather phrase "We recommend that you do not ..." (i.e. it's possible, but results in cumbersome UX)